[2023-04-26T08:41:35.986+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-04-26T08:41:34.475055+00:00 [queued]>
[2023-04-26T08:41:35.993+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: user_processing.create_table manual__2023-04-26T08:41:34.475055+00:00 [queued]>
[2023-04-26T08:41:35.994+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T08:41:35.994+0000] {taskinstance.py:1363} INFO - Starting attempt 1 of 1
[2023-04-26T08:41:35.995+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-04-26T08:41:36.003+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_table> on 2023-04-26 08:41:34.475055+00:00
[2023-04-26T08:41:36.009+0000] {standard_task_runner.py:55} INFO - Started process 588 to run task
[2023-04-26T08:41:36.015+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'user_processing', 'create_table', 'manual__2023-04-26T08:41:34.475055+00:00', '--job-id', '203', '--raw', '--subdir', 'DAGS_FOLDER/user_processing.py', '--cfg-path', '/tmp/tmp_2l9ozr1']
[2023-04-26T08:41:36.018+0000] {standard_task_runner.py:83} INFO - Job 203: Subtask create_table
[2023-04-26T08:41:36.092+0000] {task_command.py:376} INFO - Running <TaskInstance: user_processing.create_table manual__2023-04-26T08:41:34.475055+00:00 [running]> on host c452347a64f2
[2023-04-26T08:41:36.145+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=user_processing
AIRFLOW_CTX_TASK_ID=create_table
AIRFLOW_CTX_EXECUTION_DATE=2023-04-26T08:41:34.475055+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2023-04-26T08:41:34.475055+00:00
[2023-04-26T08:41:36.154+0000] {base.py:71} INFO - Using connection ID 'postgres' for task execution.
[2023-04-26T08:41:36.159+0000] {sql.py:315} INFO - Running statement: 
            CREATE TABLE IF NOT EXISTS users(
                first_name varchar(255) not null,
                last_name varchar(255) not null,
                country varchar(255) not null,
                username varchar(255) not null,
                password varchar(255) not null,
                email varchar(255) not null
            );
        , parameters: None
[2023-04-26T08:41:36.160+0000] {postgres.py:96} INFO - NOTICE:  relation "users" already exists, skipping

[2023-04-26T08:41:36.168+0000] {taskinstance.py:1406} INFO - Marking task as SUCCESS. dag_id=user_processing, task_id=create_table, execution_date=20230426T084134, start_date=20230426T084135, end_date=20230426T084136
[2023-04-26T08:41:36.187+0000] {local_task_job.py:164} INFO - Task exited with return code 0
[2023-04-26T08:41:36.207+0000] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
